code-of-conduct.md

```diff
@@ -41,6 +41,20 @@ This Code of Conduct applies within all community spaces, and also applies when
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the community leaders responsible for enforcement at conduct@confidentialcomputing.io. All complaints will be reviewed and investigated promptly and fairly.
 
+Members of the Governing Board on the reflector are: 
+Company | Representative | Email Address
+Alibaba | Xiaoning Li | xiaoning.li@alibaba-inc.com
+ARM | Philippe Robin | philippe.robin@arm.com
+Facebook | Michael Cheng | mfc@fb.com
+Google | Nelly Porter | nellyporter@google.com
+Huawei | Peixin Hou | peixin.hou@huawei.com
+Intel | Ron Perez | ronald.perez@intel.com
+Microsoft | Stephen Walli | stephen.walli@microsoft.com
+Oracle | Sergio Leunissen | sergio.leunissen@oracle.com
+Red Hat | Mike Bursell | mbursell@redhat.com
+TAC Chair | Dave Thaler | dthaler@microsoft.com
+General Member Rep | Richard Searle  | richard.searle@fortanix.com
+
```

Comment:
Ask the PR to be updated to include Accenture as well.

Reply:
@mwklein can do, shall I add your email?

Reply:
Yes, my email can be included.